.github/workflows/orca.yml

```diff
@@ -57,29 +58,31 @@ jobs:
           - INTEGRATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_DEV
         php-version: [ "8.1" ]
         orca-enable-nightwatch: [ "FALSE" ]
+        orca-coverage-enable: [ "FALSE" ]
         include:
-          - orca-job: STATIC_CODE_ANALYSIS
-            php-version: "8.1"
-            orca-enable-nightwatch: "FALSE"
-
-          - orca-job: INTEGRATED_TEST_ON_OLDEST_SUPPORTED
-            php-version: "7.4"
-            orca-enable-nightwatch: "FALSE"
-
-          - orca-job: INTEGRATED_TEST_ON_OLDEST_SUPPORTED
-            php-version: "8.0"
-            orca-enable-nightwatch: "FALSE"
-
-          - orca-job: ISOLATED_TEST_ON_NEXT_MAJOR_LATEST_MINOR_BETA_OR_LATER
+          # Testing Drupal 10 in php 8.1 with nightwatch.
+          - orca-job: ISOLATED_TEST_ON_CURRENT
             php-version: "8.1"
             orca-enable-nightwatch: "TRUE"
-
-          - orca-job: INTEGRATED_TEST_ON_LATEST_LTS
-            php-version: "7.4"
-            orca-enable-nightwatch: "FALSE"
-
```

Comment:
It would improve readability if you would put this blank lines back between jobs.

Reply:
Done